src/net/sourceforge/kolmafia/maximizer/CheckedItem.java

```diff
@@ -62,16 +62,18 @@ public CheckedItem( int itemId, int equipScope, int maxPrice, int priceLevel )
 	{
 		super( itemId, 1, false );
 
+		this.inventory = InventoryManager.getCount( itemId );
+
+		this.initial = InventoryManager.getAccessibleCount( itemId );
+
 		// special case used to get a CheckItem that .equals( EquipmentRequest.UNEQUIP ).
```

Comment:
It feels weird using itemId before checking it, but I recognize the idiom so no change needed.